pkg/virt-controller/services/template.go

```diff
@@ -357,7 +357,7 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		},
 	})
 
-	if util.IsSRIOVVmi(vmi) || util.IsGPUVMI(vmi) {
+	if util.IsGPUVMI(vmi) {
```

Comment:
for GPU we still need ?

Reply:
Not sure. but out of the scope of this PR I Think.

Reply:
sure np, but wonder if we want to address it in the future